src/components/atoms/ScrollView/Component.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { forwardRef, FunctionComponent } from 'react';
+import { ScrollView as NativeScrollView, ScrollViewProps as NativeScrollViewProps } from 'react-native';
+import type { PropsWithRef } from '../../../type';
+
+export type ScrollViewProps = PropsWithRef<NativeScrollViewProps>;
+
+const ScrollView: FunctionComponent<ScrollViewProps> = forwardRef<any, ScrollViewProps>(
```

Comment:
It seem it should be

```suggestion
const ScrollView: FunctionComponent<ScrollViewProps> = forwardRef<NativeScrollView, ScrollViewProps>(
```